dbclient/dbclient/src/main/java/io/helidon/dbclient/DbClient.java

```diff
@@ -63,6 +64,19 @@
      */
     String dbType();
 
+    /**
+     * Unwrap database client internals.
+     * Only database connection is supported. Any operations based on this connection are <b>blocking</b>.
+     * Reactive support must be implemented in user code.
+     * Mongo database driver internals support {@code MongoClient} and {@code MongoDatabase} classes.
```

Comment:
Probably better to avoid the reference to impls since they are not links.

Reply:
OK, I'm removing it. But API user won't know what classes are supported as unwrap targets now.

Reply:
In that case you could create a class like `DbClients` and have constants for class names.